invoices_test.go

```diff
@@ -123,6 +124,7 @@ func TestInvoices_List(t *testing.T) {
 		UUID:             "421f7b7d414e4c6792938e7c49d552e9",
 		State:            recurly.InvoiceStateOpenDeprecated,
 		InvoiceNumber:    1005,
+		DiscountInCents:  0,
```

Comment:
Let's put an amount in here so that we know the library is unmarshaling it correctly.